sys/include/net/ng_fib.h

```diff
@@ -34,6 +34,14 @@ typedef struct rrp_address_msg_t {
     uint32_t address_flags; /**< The flags for the given address */
 } rrp_address_msg_t;
 
+/**
+ * @brief entry used to collect available destinations
+ */
+typedef struct fib_destination_set_entry_t {
+    uint8_t dest[16];    /**< The destination address */
```

Comment:
@BytesGalore I'm in favor of getting this PR merged asap. I think the fixes to your address_compare functions are significant. Magic numbers can be replaced after #3022